components/application-mgt/org.wso2.carbon.identity.application.mgt/pom.xml

```diff
@@ -70,6 +70,12 @@
         <dependency>
             <groupId>org.wso2.carbon</groupId>
             <artifactId>org.wso2.carbon.user.core</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
Wonder why this is needed..

Reply:
Junit dependency is coming from this component and had a hard time in resolving the testng component. Technically this is no mandatory if we have the correct dependency order.